modules/ROOT/pages/dataweave-formats.adoc

```diff
@@ -738,54 +681,73 @@ See xref:6@studio::input-output-structure-transformation-studio-task.adoc[To Def
 
 
 
-== Cobol Copybook
+== COBOL Copybook
 
-Copybook types are technically considered a type of <<Flat File>> format, but when selecting this option the Transform Message component offers you settings that are better tailored to the needs of this format.
+Copybook types are technically considered a type of <<Flat File>> format, but
+when you select this option, the Transform Message component offers you settings
+that are better tailored to the needs of the COBOL Copybook format.
 
 [[cobol_metadata]]
 === Importing a Copybook Definition
 
+When you import a Copybook definition, the Transform Message component converts
+the definition to a flat file schema that you can reference with `schemaPath`
+property.
+
 To import a copybook definition:
 
-. Click Define Metadata on the input Payload in the Transform component, and select Set Metadata to open the Set Metadata Type dialog.
+. Right-click the input payload in the Transform component in Studio, and select
+Set Metadata to open the Set Metadata Type dialog.
```

Comment:
Should the button name be surrounded by asterisks since it is a GUI component?

Reply:
done